src/state.rs

```diff
@@ -0,0 +1,30 @@
+pub(crate) mod game;
+
+use notan::prelude::*;
+
+use game::{defeat, playing, victory, Stage};
+
+#[derive(AppState)]
```

Comment:
`#[derive(AppState)]`
This derives from Notan specific struct?

Reply:
Close, it derives the `AppState` marker trait so that `State` can be passed as parameter to the `update` functions